asian-journal-of-neurosurgery.csl

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>Asian Journal of Neurosurgery</title>
+    <id>http://csl.mendeley.com/styles/490764671/asian-journal-of-neurosurgery</id>
+    <link href="http://www.zotero.org/styles/neurology-india" rel="self"/>
```

Comment:
Line 5 and 6.
ID should be with www.zotero.org and not Mendeley.
Self link also not correct. Still says Neuro India.